eZ/Bundle/EzPublishCoreBundle/Command/ReindexCommand.php

```diff
@@ -207,7 +235,7 @@ protected function indexIncrementally(InputInterface $input, OutputInterface $ou
         }
 
         $iterations = ceil($count / $iterationCount);
-        $processes = $input->getOption('processes');
+        $processes = $continueOnError ? 1 : $input->getOption('processes');
```

Comment:
If `continue-on-error` doesn't allow for `processes` setting, it should be stated in a command doc also.

Reply:
I need to check again whether with this new approach this is still relevant. If yes, then I'll add a note to command docs.

Reply:
I've tested it, and it's not going to work with parallel processing even with this new approach.
 
The information which you've asked for is currently in the command option description:
https://github.com/ezsystems/ezpublish-kernel/pull/2221/files#diff-81257ebfd63be05261b508add93d35edR137
It is enough, isn't it? Or would you like to add it also to command help, perhaps?

Reply:
I would also be good to update `processes` option help stating that it cannot be used with `continue-on-errors`.

Reply:
Added in https://github.com/ezsystems/ezpublish-kernel/pull/2221/commits/91594d961baf39a4937c03dff479b64595de611d.